yrand/common.go

```diff
@@ -7,19 +7,44 @@ import (
 	"math/big"
 )
 
+var (
+	// QuitShuffle is used as a return value from ShuffleIndexFunc to indicate that the execution of WeightedShuffle should be terminated immediately.
+	// It is not returned as an error by any function.
+	QuitShuffle = errors.New("quit this shuffle")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [error var QuitShuffle should have name of the form ErrFoo](https://app.codacy.com/manual/an9an63/gut/pullRequest?prid=5011154)